openstack/dns/v2/recordsets/results.go

```diff
@@ -93,12 +93,16 @@ type RecordSet struct {
 	// Status is the status of the recordset.
 	Status string `json:"status"`
 
+	Default bool `json:"default"`
+
+	// Deprecated
```

Comment:
please put the deprecated fields together.

Reply:
Ok, Please check it again.